store.go

```diff
@@ -3522,12 +3522,12 @@ func ReloadConfigurationFile(configFile string, storeOptions *StoreOptions) {
 		fmt.Printf("Failed to parse %s %v\n", configFile, err.Error())
 		return
 	}
-	if os.Getenv("STORAGE_DRIVER") != "" {
-		config.Storage.Driver = os.Getenv("STORAGE_DRIVER")
-	}
 	if config.Storage.Driver != "" {
 		storeOptions.GraphDriverName = config.Storage.Driver
 	}
+	if os.Getenv("STORAGE_DRIVER") != "" {
+		config.Storage.Driver = os.Getenv("STORAGE_DRIVER")
```

Comment:
👍

Reply:
This does not change anything, as far as I can see.


In the old code config.Storage.Driver will be set to os.Getenv("STORAGE_DRIVER") if it was set, and then the 
if config.Storage.Driver will not == "".

Reply:
@rhatdan ah thanks, i didn't understand the code properly, would setting `storeOptions.GraphDriverName` instead works ?

Reply:
No I am fine with the change from a readability point of view. but I wanted to know if someone found a real bug, because I don't believe this fixes anything.

Reply:
The bug is when you set the environment variable `STORAGE_DRIVER`, it doesn't take precedence on the configuration file, is it expected? 

`STORAGE_OPTS`  seems to works differently where the STORAGE_OPTS env are added to the already storage opts set from the file  https://github.com/containers/storage/blob/3ef97f424/store.go#L3600-L3604

Reply:
If that is true, then it will still be true after this PR, since it changes nothing.

Reply:
I misread the original code.  With this change `storeOptions.GraphDriverName` won't be set, so I think the old code is correct